react/tic-tac-toe/src/app/screens/Login/styles.module.scss

```diff
@@ -17,5 +18,12 @@
   font-size: 16px;
   padding: 14px 40px;
   border: 1px solid $green;
+  border-radius: 3px;
   background-color: $light-green;
+  margin-top: 30px;
+}
+
+.buttonSubmit:hover{
```

Comment:
`button-submit:hover`

Reply:
I've made these corrections on a new branch that i haven pushed yet